apps/test/unit/blockly/addons/cdoFieldDropdownTest.js

```diff
@@ -0,0 +1,94 @@
+import {__TestInterface} from '@cdo/apps/blockly/addons/cdoFieldDropdown';
+import {expect} from '../../../util/reconfiguredChai';
+
+describe('Testing CdoFieldDropdown function - getUpdateOptionsFromConfig', () => {
```

Comment:
Nit: these descriptions are used when tests fail, so I think we know they're tests :). I also think the fact that this tests cdoFieldDropdown is apparent from the file name. I'd suggest naming this either the function you're testing (`getUpdatedOptionsFromConfig`), or something descriptive about what this function does ( think your later blocks of tests do a good job of this, so maybe just the function name?)

Reply:
That makes sense - will just name the function. Thanks!